seed/serializers/analysis_property_views.py

```diff
@@ -8,10 +8,12 @@
 
 from seed.models import AnalysisPropertyView
 from seed.serializers.analysis_output_files import AnalysisOutputFileSerializer
+from seed.serializers.properties import PropertyStateSerializer
 
 
 class AnalysisPropertyViewSerializer(serializers.ModelSerializer):
     output_files = AnalysisOutputFileSerializer(source='analysisoutputfile_set', many=True)
+    property_state = PropertyStateSerializer()
```

Comment:
Nice! Great to see adding PropertyState info is as easy as doing this. Note that displaying this info on the frontend is beyond MVP scope, but it's definitely something we want at some point.

Reply:
This was added so the address could be displayed.  Is there a better way to grab that?

Reply:
ah - forgot about that. Nope this is ideal since, we'll end up needing everything later! 👌